src/rrt/Tree.hpp

```diff
@@ -52,7 +68,10 @@ class Node {
      */
     const T& state() const { return _state; }
 
+    double* coordinates() { return _vec.data(); }
```

Comment:
@ashaw596 Is `double*` a good return here or would returning the `std::vector` be better?